backend-implicaction/src/main/java/com/dynonuggets/refonteimplicaction/dto/JobPostingDto.java

```diff
@@ -22,7 +22,7 @@
     private String salary;
     private String keywords;
     private ContractTypeDto contractType;
-    private StatusDto status;
     private Instant createdAt;
+    private boolean isArchive;
```

Comment:
tu peux l'appeler `archive`stp. lombock qu'on utilise pour la génération des getters et setters va générer une méthode qu'il va nommer 'is' + 'nom de l'attribut' dans le cas d'un attribut de type booléen ... ça fait bizarre du coup d'avoir un jobPostingDto.isIsArchive().

```suggestion
    private boolean archive;
```